src/Juvix/Data/Fixity.hs

```diff
@@ -68,13 +70,13 @@ isUnary :: Fixity -> Bool
 isUnary = not . isBinary
 
 appFixity :: Fixity
-appFixity = Fixity PrecApp (Binary AssocLeft)
+appFixity = Fixity PrecApp (Binary AssocLeft) Nothing
 
 funFixity :: Fixity
-funFixity = Fixity PrecArrow (Binary AssocRight)
+funFixity = Fixity PrecArrow (Binary AssocRight) Nothing
 
 updateFixity :: Fixity
-updateFixity = Fixity PrecUpdate (Unary AssocPostfix)
+updateFixity = Fixity PrecUpdate (Unary AssocPostfix) Nothing
```

Comment:
Change this to follow the pattern `= Fixity {_fixityPrecedence = .., _fixityArity=..`.

Reply:
done